### PR TITLE
Refactor (some) activity functional test cases

### PR DESCRIPTION
**Pull request by tarnap**

### CODE REVIEW

**Comment by Rotonen** [comment]
Hijacked and polished this one.

Ready for review.

File: opengever/activity/tests/test_resolve.py [hunk 1]
```diff
@@ -1,125 +1,85 @@
...
-            browser.login().open(
-                self.portal, view='resolve_notification',
-                data={'notification_id': notification.notification_id})
+            self.login(self.regular_user, browser)
+            browser.open(self.portal, view='resolve_notification', data={'notification_id': notification.notification_id})
+
+    @browsing
+    def test_mark_notification_as_read(self, browser):
+        self.login(self.regular_user, browser)
+        # XXX - We have to hit the view twice here to mark it as read - a timing issue or a bug?
```
Comment (phgross) [inline opengever/activity/tests/test_resolve.py]: Re fetching the notification should solve your issue. Should definitely not be necessary to call the view twice.

Comment (Rotonen) [reply opengever/activity/tests/test_resolve.py]: As `notification_center().get_notification(self.notification.notification_id).is_read` also returns `False` when the view is hit only once, which is how I ended up trying this out. I'm very unsure as to what is going on there.

Comment (phgross) [reply opengever/activity/tests/test_resolve.py]: Seems weird/wrong i'll take a look into it tomorrow ...

**Comment by Rotonen** [comment]
@phgross so are we now at the end of our hijack tennis here?

**Pull request merged**